### ISSUE TITLE
How do you scrape messages from here?

### ISSUE THREAD

**Issue by phr34k0** [enjoy2000/discord-pokemon-go-snipe-bot#61]
http://pogo-feed.mmoex.com/messages

**Comment by Navyguy330** [comment]
yes please use this one,  it also has IV and seems to update on a live stream

**Comment by phr34k0** [comment]
This crap is x100 better then this python crap that was coded. It pulls everything from public channel. The current coded is shit. Doesn't even work half the time

[
  {
    "id": "209314780870213633",
    "Server": "Necrobot.io",
    "Name": "plus_90_iv"
  },
  {
    "id": "209314800935763977",
    "Server": "Necrobot.io",
    "Name": "100_iv"
  },
  {
    "id": "209283402245865472",
    "Server": "Necrobot.io",
    "Name": "sniping"
  },

  {
    "id": "207998375251935232",
    "Server": "PokeMobBot - a Pokémon Go Bot",
    "Name": "coordsbots"
  },
  {
    "id": "208936978186502145",
    "Server": "PokeMobBot - a Pokémon Go Bot",
    "Name": "90_plus_iv"
  },


  {
    "id": "209260366784495616",
    "Server": "PokéGO- RARE SPOT & Chat!",
    "Name": "rare_spotting"
  },
  {
    "id": "209680517740756994",
    "Server": "PokéGO- RARE SPOT & Chat!",
    "Name": "snipebot_coords"
  },
  {
    "id": "209171120702619648",
    "Server": "PokéGO- RARE SPOT & Chat!",
    "Name": "90plus_ivonly"
  },
  {
    "id": "209174231269769216",
    "Server": "PokéGO- RARE SPOT & Chat!",
    "Name": "donor_100iv"
  },


  {
    "id": "208856384744194048",
    "Server": "PokemonGo Rare Hunting",
    "Name": "coord-bot"
  },
  {
    "id": "207092045683097600",
    "Server": "PokemonGo Rare Hunting",
    "Name": "high_iv"
  },
  {
    "id": "207247439004958720",
    "Server": "PokemonGo Rare Hunting",
    "Name": "rare_pokemon"
  },
  {
    "id": "206238613040660490",
    "Server": "PokemonGo Rare Hunting",
    "Name": "other_pokemon"
  },


  {
    "id": "208754684414918656",
    "Server": "100IV CLUB",
    "Name": "95-iv-and-up"
  },
  {
    "id": "209077347503046656",
    "Server": "100IV CLUB",
    "Name": "rare-coords-bot "
  },
  {
    "id": "208752256999227393",
    "Server": "100IV CLUB",
    "Name": "100-iv "
  },
  {
    "id": "208836707057926144",
    "Server": "100IV CLUB",
    "Name": "rare-pokemon "
  }

]
